cypress/integration/login.spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Test For Login Page and Forgot Password', function() {
+	context('Login Page', function() {
+		beforeEach(function() {
+			cy.visit('localhost:3000/login');
+        })
+        it('test login no input fileds',function(){
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
+        });
+        it('test login valid email',function(){
+            cy.get('input[name="email"]').type('manupanday1998@gmail.com').should('have.value', 'manupanday1998@gmail.com')
+            cy.get('input[name="password"]').type('123456').should('have.value', '123456')
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
+        });
+        it('test login invalid email',function(){
+            cy.get('input[name="email"]').type('sagmail.com').should('have.value', 'sagmail.com')
+            cy.get('input[name="password"]').type('retro').should('have.value', 'retro')
+            cy.get('.button.is-success').click()
+            cy.url().should('include', '/')
```

Comment:
here /login

Reply:
since i am already on /login using '/' gives the desired result,

Reply:
I will still resolve for more clarity

Reply:
yeah but if website, routes to /home for because of some bug , this will pass but we want it to fail right